packages/design-system/src/components/FormLabel/FormLabel.jsx

```diff
@@ -50,14 +50,24 @@ export class FormLabel extends React.PureComponent {
   }
 
   render() {
-    const { fieldId, id, children, textClassName, className, inversed } = this.props;
+    const {
+      fieldId,
+      id,
+      children,
+      textClassName,
+      className,
+      inversed,
+      errorMessage,
+      requirementLabel,
+      ...labelProps
+    } = this.props;
```

Comment:
we want to define all of our props here including `component` and hint` to ensure only extra props get passed to `labelProps`

Reply:
Thank you!